static/js/calculate_score.js

```diff
@@ -359,6 +391,141 @@ var calculatePolyScore = async () => {
     }
 }
 
+/**
+ * 
+ * @param {*} gwasUploadFile the uploaded tab separated GWAS data file
+ * @param {*} gwasRefGen the reference genome of the GWAS upload data
+ * @param {*} refGen the reference genome of the samples
+ * @returns the associations data object used in calculating polygenic risk scores
+ */
+async function getGWASuploadData(gwasUploadFile, gwasRefGen, refGen) {
+    var fileContents = await readFile(gwasUploadFile);
+    fileLines = fileContents.split("\n")
+
+    //TODO? put in that they have the option to include other columns and we will add that to the output??
```

Comment:
Don't forget this todo. Are their specific columns you think they might add? I don't think this is necessary for us to do, unless you have a good example.

Reply:
I already added in support for optional citation and reported trait columns. This todo will be removed